### ISSUE TITLE
DropdownList doesn't search items above the selected value

### ISSUE THREAD

**Issue by jedwards1211** [jquense/react-widgets#596]
To repro, in the demo example:

1. Click the dropdown and type in `lucile`
2. Hit enter to select `Lucile Williams`
3. Click the dropdown and type in `gilbert`
4. The dropdown should outline `Gilbert Diaz` but it doesn't
5. Type in `oscar`
6. The dropdown outlines `Oscar Diaz`

**Comment by jquense** [comment]
yeah it starts searching at the focused item, it should loop back around tho

**Comment by rokborf** [comment]
Any updates here?

**Comment by jquense** [comment]
updates to what? the question was answered

**Comment by jedwards1211** [comment]
@jquense Oh, I thought by

> it should loop back around tho

you meant this is a bug and you still need to make it loop back around.  Are you saying you believe the existing code already loops back around?  It seemed to me like it wasn't doing that.

**Comment by jquense** [comment]
you're right! sorry yes that is a bug
